### PR TITLE
[13.0][MIG] account invoice refund reason: Migration to 13

### PR DESCRIPTION
**Pull request by JaumeBforgeFlow**
Migration to v13 using as reference #1138 (Migration to v14)

### CODE REVIEW

File: account_invoice_refund_reason/tests/test_account_invoice_refund_reason.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                    "account_id": self.account_id.id,
+                    "name": "product test 5",
+                    "price_unit": 100.00,
+                },
+            )
+        ]
+        self.account_invoice_customer0 = self.account_invoice_obj.create(
+            dict(
+                name="Test Customer Invoice",
+                move_type="out_invoice",
```
Comment (JuanyDForgeflow) [inline account_invoice_refund_reason/tests/test_account_invoice_refund_reason.py]: change to type

**Comment by MiquelRForgeFlow** [comment]
Add commit history please.

**Comment by JaumeBforgeFlow** [comment]
@MiquelRForgeFlow  @JuanyDForgeflow fixed!

File: account_invoice_refund_reason/tests/test_account_invoice_refund_reason.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        )
+
+    def test_onchange_reason_id(self):
+        self.account_invoice_customer0.action_post()
+        self.account_invoice_refund_0 = self.invoice_refund_obj.with_context(
+            active_model="account.move", active_ids=self.account_invoice_customer0.ids
+        ).create(
+            dict(
+                refund_method="refund",
+                date_mode="custom",
```
Comment (MiquelRForgeFlow) [inline account_invoice_refund_reason/tests/test_account_invoice_refund_reason.py]:
```suggestion
                date="custom",
```

File: account_invoice_refund_reason/i18n/es.po [hunk 3]
```diff
@@ -0,0 +1,130 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* account_invoice_refund_reason
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 14.0+e\n"
```
Comment (MiquelRForgeFlow) [inline account_invoice_refund_reason/i18n/es.po]:
```suggestion
"Project-Id-Version: Odoo Server 13.0+e\n"
```

File: account_invoice_refund_reason/wizard/account_move_reversal.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMoveReversal(models.TransientModel):
+    _inherit = "account.move.reversal"
+
+    reason_id = fields.Many2one(
+        "account.invoice.refund.reason", string="Reason to credit"
```
Comment (GuillemCForgeFlow) [inline account_invoice_refund_reason/wizard/account_move_reversal.py]:
```suggestion
        "account.invoice.refund.reason", string="Refund Reason"
```
Perhaps using the same name in the wizard as the one in the view just to not confuse the user.

**Review by GuillemCForgeFlow — CHANGES_REQUESTED** [review]

**Comment by JaumeBforgeFlow** [comment]
@GuillemCForgeFlow @MiquelRForgeFlow Fixed!

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by GuillemCForgeFlow — APPROVED** [review]
:+1:

**Review by JordiMForgeFlow — APPROVED** [review]
Code + functional review 👍🏼

**Review by AaronHForgeFlow — APPROVED** [review]

**Pull request merged**